### ISSUE TITLE
Merge upstream "eth" branch and dashify

### ISSUE THREAD

**Issue by UdjinM6** [deginner/lamassu-server#9]
Some work has already been done there https://github.com/lamassu/lamassu-server/tree/eth. Even though it's not yet complete (or at least that was my impression) I guess it might be a good idea to follow the same path (and maybe even contribute back) instead of trying to implement some separate solution.

[EDIT]
Changed to reflect definitively that this task needs to be done. Specifically, this repository needs to be changed to reflect the exchange_config schema that the upstream eth branch uses. Also, the improvements in configuration made available via the command line interface should be adapted to allow Dash configuration. Basically, merge everything from upstream eth and eth-notification, then make sure it runs with Dash. Overwrite local changes when necessary.
[/EDIT]

**Comment by isysd** [comment]
I'm aware of this and other work the Lamassu team is doing, but they've indicated none of it is ready. Will have their cooperation in merging the projects when they're ready. :/

**Comment by isysd** [comment]
Still have no definitive statement on the matter, but the eth and eth-notification upstream branches have a number of changes that seem to be settling in. Notably, they chose a different database structure than the Dash implementation, and also developed a new CLI configuration system.

**Comment by UdjinM6** [comment]
Yep, noted https://github.com/lamassu/lamassu-server/tree/eth-notification too.Haven't time to look at this closely but I guess https://github.com/lamassu/lamassu-server/tree/eth-notification have all of https://github.com/lamassu/lamassu-server/tree/eth and just continues from where eth branch stopped.

**Comment by isysd** [comment]
@cindy-zimmerman

**Comment by isysd** [comment]
# Git Guild Bounty Program

| Total | Dev               | Sales | Reporter | Tester(s) | Maintainer | Documenter |
|--------|------------------|---------|-------------|------------|-----------------|----------------|
|N/A    | assignee(s)   | @isysd  | @isysd | @isysd | @isysd  | @d3brouille |
| $1,833.33 | $1,100.00 | $91.67 | $91.67 | $366.67 | $91.67 | $91.67 |
